lib/elixir/lib/calendar/date.ex

```diff
@@ -176,6 +176,23 @@ defmodule Date do
     calendar.days_in_month(year, month)
   end
 
+  @doc """
+  Returns the number of days in the given `date` month.
```

Comment:
Shouldn't it be "Returns the number of months in the given year"?

Reply:
Good catch @David-Gil, fixed in 7d621f60a9bd50a03f8eca8fc524aa7d041fa8f1 💟